spec.bs

```diff
@@ -380,14 +380,14 @@ To get the <dfn>max batch size</dfn> for an [=/origin=] |issuer|, run the follow
 
 To <dfn>generate masked tokens</dfn> given [=key commitments=] |issuerKeys| and number |numTokens|, run the following steps:
 
-1. Let |maskOperation| be the Mask operation corresponding to |issuerKeys|["protocol_version"].
+1. Let |maskOperation| be the Mask operation corresponding to |issuerKeys|[cryptoProtocolVersion].
```

Comment:
That doesn't seem correct, did you rename the JSON field to `"cryptoProtocolVersion"`? If so, please rename it in https://wicg.github.io/trust-token-api/#issuer-public-keys and include quotes here.

Reply:
Fixed.